chart/values.yaml

```diff
@@ -675,6 +675,8 @@ nats:
   cluster:
```

Comment:
`hostNetwork: true
dnsPolicy: ClusterFirstWithHostNet`

Instead can we add this config to Nats to get the domain by default? When we add this config here, it is added to the Nats pods spec by default. We are doing similarly for io-engine daemon set spec as well.